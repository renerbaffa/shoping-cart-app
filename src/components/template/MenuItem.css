.MenuItem-container {
  cursor: pointer;
}

.MenuItem-container:hover {
  cursor: pointer;
  color: gray;
}
