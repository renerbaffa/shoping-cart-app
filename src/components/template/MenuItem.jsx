import React from 'react';
import PropTypes from 'prop-types';
import cx from 'classnames';

import './MenuItem.css';

const MenuItem = ({ className, label }) => (
  <div className={cx('MenuItem-container', className)}>
    {label}
  </div>
);

MenuItem.propTypes = {
  className: PropTypes.string,
  label: PropTypes.string,
};

MenuItem.defaultProps = {
  className: '',
  label: '',
};

export default MenuItem;
