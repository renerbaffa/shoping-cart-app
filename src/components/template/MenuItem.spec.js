import React from 'react';
import renderer from 'react-test-renderer';

import MenuItem from './MenuItem';

describe('<MenuItem />', () => {
  it('should show spinner correctly', () => {
    const menuItem = renderer.create(<MenuItem />).toJSON();
    expect(menuItem).toMatchSnapshot();
  });
})