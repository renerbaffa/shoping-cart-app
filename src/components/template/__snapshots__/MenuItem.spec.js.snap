// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuItem /> should show spinner correctly 1`] = `
<div
  className="MenuItem-container"
>
  
</div>
`;
